src/hyp3_isce2/process.py

```diff
@@ -8,9 +8,12 @@
 from argparse import ArgumentDefaultsHelpFormatter, ArgumentParser
 from pathlib import Path
 
+from hyp3lib.get_orb import downloadSentinelOrbitFile
```

Comment:
I want to ask @jhkennedy to remind me what is our ultimate goal for https://github.com/ASFHyP3/hyp3-lib. Would we eventually like to eliminate it as a dependency entirely and archive the repo, or are we still fine with using it internally at ASF?

Reply:
I think we have long-term plans to either sunset or significantly downsize hyp3-lib to only the useful utilizes but I think we're a ways off from have the time to commit to this effort.

Reply:
Sounds good.